packages/xfsprogs/Cargo.toml

```diff
@@ -0,0 +1,26 @@
+[package]
+name = "xfsprogs"
+version = "0.1.0"
+edition = "2021"
+publish = false
+build = "../build.rs"
+
+[lib]
+path = "/dev/null"
+
+[package.metadata.build-package]
+releases-url = "https://xfs.wiki.kernel.org"
+
+[[package.metadata.build-package.external-files]]
+url = "http://kernel.org/pub/linux/utils/fs/xfs/xfsprogs/xfsprogs-5.15.0.tar.xz"
```

Comment:
Does the latest version not work? Typically these packages are moved to the latest version as part of the update cycle, so if it doesn't work now then it'll just break for the next maintainer that comes along.

Reply:
I was having difficulties compiling with some strange errors. I've lost them since but will try moving to 6.3.0 and see if they reappear.

Reply:
I moved back to 6.3.0 and it builds and worked perfectly. I think it might have been other things stabilizing with liburcu so it seems fine now.